cypress/e2e/formr-b/FormRB.spec.ts

```diff
@@ -211,30 +204,30 @@ describe("Form R (Part B) - desktop", () => {
       dayjs(farFutureDate).format("DD/MM/YYYY")
     );
     // check if Covid section exists or not depending on flag
-    if (isCovid) {
-      cy.get("[data-cy=sectionHeader7]").should("exist");
-      cy.get("[data-cy=BtnEditSection7]").should("exist");
-      cy.get("[data-cy=educationSupervisorName]").should(
-        "contain.text",
-        "No supervisor name provided"
-      );
-      cy.get("[data-cy=educationSupervisorEmail]").should(
-        "contain.text",
-        "No supervisor email provided"
-      );
-    } else {
-      cy.get("[data-cy=sectionHeader7]").should("not.exist");
-      cy.get("[data-cy=BtnEditSection7]").should("not.exist");
-    }
+    // if (isCovid) {
```

Comment:
@EdwardBarclay Not sure why this code is commented out. Doesn't the if/else statement do its job? What happened when you left this code uncommented and ran the test locally? I'm assuming isCovid is set to false for this test?